### DISCUSSION TITLE
Show special characters on tabby

### DISCUSSION THREAD

**Discussion by Kania78** [Eugeny/tabby#8111] [Q&A]
Hi,
I would like help with the configuration of the TABBY terminal
I would like to display the special characters: '?', 'è', 'é'.
Can you tell me how to configure Tabby to achieve this please?
Thanks in advance
![image](https://user-images.githubusercontent.com/128031946/226564739-d896887f-8e76-4d35-8736-01f64ee15fdd.png)

**Comment by PAIMPOL** [comment]
Hello,
i have same problems, I change the fonts but it doesn't change anything.
Thanks for help.

**Comment by PAIMPOL** [comment]
![image](https://user-images.githubusercontent.com/128511503/227447931-8d0ea012-b547-4a4a-ac3e-220c559ac666.png)
